cloudformation/curprocessor-sam.yml

```diff
@@ -64,14 +207,40 @@ Resources:
       Timeout: 30
       MemorySize: 128
       CodeUri: ../
-      Fn::Transform:
-        Name: AWS::Include
-        Parameters:
-          Location : !Ref EnvConfigFileS3Uri
+      Events:
+        Manifests:
+          Type: S3
+          Properties:
+            Bucket: !Ref CostUsageReport
+            Events: s3:ObjectCreated:*
+            Filter:
+              S3Key:
+                Rules:
+                  - Name: prefix
+                    Value: aws-reports/
```

Comment:
This should be a reference to ReportPathPrefix (!Sub '${ReportPathPrefix}/'

Reply:
Ah totally missed that one, added it thanks!